.github/workflows/continous-integration.yml

```diff
@@ -1,198 +0,0 @@
-name: Continous Integration
```

Comment:
This workflow was renamed on `main` (there was a typo in the name) but the `documentation` branch wasn't cleaned up.